mathics/builtin/specialfns/gamma.py

```diff
@@ -438,14 +447,27 @@ class Pochhammer(SympyFunction):
     attributes = A_LISTABLE | A_NUMERIC_FUNCTION | A_PROTECTED
 
     rules = {
-        "Pochhammer[0, n_]": "0",  # Wikipedia says it should be 1 though.
+        "Pochhammer[0, 0]": "1",
         "Pochhammer[a_, n_]": "Gamma[a + n] / Gamma[a]",
         "Derivative[1,0][Pochhammer]": "(Pochhammer[#1, #2]*(-PolyGamma[0, #1] + PolyGamma[0, #1 + #2]))&",
         "Derivative[0,1][Pochhammer]": "(Pochhammer[#1, #2]*PolyGamma[0, #1 + #2])&",
     }
     summary_text = "compute Pochhammer's symbols"
     sympy_name = "RisingFactorial"
 
+    def eval_negative_int(self, n, evaluation: Evaluation):
+        "Pochhammer[0, n_?Negative]"
+
+        n_value = n.value
+        if isinstance(n, (MachineReal, PrecisionReal)):
+            if n_value == int(n_value):
+                n_value = int(n_value)
+            else:
+                return MachineReal0
+
+        sympy_result = tracing.run_sympy(sympy.rf, 0, n.value)
```

Comment:
@aravindh-krishnamoorthy in a prior commit suggestion in your recent draft PR, I had suggested using `run_mpmath` and by extension `run_sympy`. 

However I had given the wrong version of `run_mpath`: we want the function from in `mathics.eval.tracing` not ` mathics.evalarithmetic`. And one needs to use `tracing.run_xxx` so that we can hook in property. 

In that draft PR, please make sure to use `trace.run_sympy` and `trace.run_mpmath` in this fashion.  Thanks.